src/components/Navbar/SettingsDrawer/WordByWordSection.tsx

```diff
@@ -62,6 +58,33 @@ const WordByWordSection = () => {
 
   const wordByWordFontScale = useSelector(selectWordByWordFontScale, shallowEqual);
 
+  const [hasError, setHasError] = useState(false);
+  const [translations, setTranslations] = useState<AvailableWordByWordTranslation[]>([]);
+
+  useEffect(() => {
+    getAvailableWordByWordTranslations(lang)
+      .then((res) => {
+        if (res.status === 500) {
+          setHasError(true);
+        } else {
+          const data = uniqueArrayByObjectProperty(res.wordByWordTranslations, 'isoCode');
+          setTranslations(data);
+        }
+      })
+      .catch(() => {
+        setHasError(true);
+      });
+  }, [lang]);
+
+  if (hasError) {
+    return null;
+  }
+
```

Comment:
Why don't we use `useSwr` which will help with caching calls on FE and less code and state handling?